content/articles/create-a-registration-form-using-react.js-and-connecting-it-with-firebase/index.md

```diff
@@ -57,11 +62,22 @@ This is a boilerplate code that comes when we create a react app. Now open the l
 ```
 
 
-This is where our code start you can change the meta tags as per your requirements.
+Now the basic requirements are completed. You may change the meta tags as per your preference.
+
 
-Now go inside the src folder there you will find `app.js` that's where we start our code. In react we write JSX for writing our HTML code. Create a new folder named components inside the src folder. Here we will make all of our components and then render them in App.js.
+#### Creating a Registration Form
```

Comment:
All main headings should be H3, and subheadings H4. Please correct this for all instances.

Reply:
top heading the main is h3 rest all are h4 as a subheadings